src/Tribe/Views/V2/Manager.php

```diff
@@ -42,6 +42,42 @@ class Manager {
 	 */
 	public static $option_mobile_default = 'mobile_default_view';
 
+	/**
+	 * Registration objects for auto-registered views.
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	private $view_registration = [];
+
+	/**
+	 * Registers a view such that sensible defaults are registered and hooked.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $slug View slug.
+	 * @param string $name View name.
+	 * @param string $class View class.
+	 * @param int $priority View registration priority.
+	 *
+	 * @return View_Register
+	 */
+	public function register_view( $slug, $name, $class, $priority = 30 ) {
```

Comment:
Should we have a `unregister`?

Reply:
Maybe some day. I don't see a need for it right now.